package.json

```diff
@@ -24,7 +24,9 @@
     "test:vrfCoordinator": "mocha --exit 'test/vrfCoordinator.test.ts'",
     "test:automationRegistrar": "mocha --exit 'test/automationRegistrar.test.ts'",
     "test:automationRegistry": "mocha --exit 'test/automationRegistry.test.ts'",
-    "test": "yarn test:prepare && yarn test:dataFeed && yarn test:dataFeedProxy && yarn test:ensFeedsResolver && yarn test:feedRegistry && yarn test:l2FeedUptimeSequencer && yarn test:vrfCoordinator && yarn test:automationRegistrar && yarn test:automationRegistry",
+    "test:functionsRouter": "mocha --exit 'test/functionsRouter.test.ts'",
```

Comment:
dont think this should be functions Router? The functions router is this:https://docs.chain.link/chainlink-functions/supported-networks#sepolia-testnet

not the consumer.